### PR TITLE
Fixed DDoc warnings

### PR DESCRIPTION
**Pull request by lionello**
Found by adding -w to Ddoc command line.

### CODE REVIEW

File: std/socket.d [hunk 1]
```diff
@@ -1905,6 +1905,8 @@ version(StdDdoc)
      */
     class UnixAddress: Address
     {
+        private this() pure nothrow @nogc {}
+
```
Comment (schveiguy) [inline std/socket.d]: These changes have already been merged, see #3344

Comment (lionello) [inline std/socket.d]: Thanks.

Comment (schveiguy) [inline std/socket.d]: You still have this in there. look here https://github.com/D-Programming-Language/phobos/pull/3349/files#diff-87a79b3c150b9493bd14afe80b5219c9L1892, this has already been added.

**Comment by andralex** [comment]
nice, thx!

**Comment by andralex** [comment]
Auto-merge toggled on

**Comment by schveiguy** [comment]
@andralex I don't think the docs will build still, because there are now 2 default ctors for `UnixAddress`...

Making a new PR...

**Comment by lionello** [comment]
Thanks @schveiguy ! I didn't get a chance to do it, so thanks for having my back ;)

**Pull request merged**